### PR TITLE
[cleanup][owasp] Supress false positive netty-tcnative

### PR DESCRIPTION
**Pull request by nicoloboschi**
### Motivation
Owasp check fails with 
```
One or more dependencies were identified with known vulnerabilities in Pulsar :: Distribution :: Server:

netty-tcnative-boringssl-static-2.0.52.Final-osx-x86_64.jar (pkg:maven/io.netty/netty-tcnative-boringssl-static@2.0.52.Final, cpe:2.3:a:chromium_project:chromium:2.0.52:*:*:*:*:*:*:*) : CVE-2011-1797

```

It's clearly a false positive https://github.com/jeremylong/DependencyCheck/issues/4776

### Modifications

* Suppress the violation

- [x] `doc-not-needed`

### CODE REVIEW

**Review by tisonkun — APPROVED** [review]
Thank you!

It seems this FP has been resolved on upstream https://github.com/jeremylong/DependencyCheck/issues/4154 but it suffers a regression.

File: src/owasp-dependency-check-false-positives.xml [hunk 1]
```diff
@@ -54,6 +54,13 @@
     <packageUrl regex="true">^pkg:maven/io\.netty/netty\-tcnative\-classes@.*$</packageUrl>
     <cpe>cpe:/a:netty:netty</cpe>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
+   file name: netty-tcnative-boringssl-static-2.0.52.Final-osx-x86_64.jar
+   ]]></notes>
+    <sha1>87a933c786d4b38355ebbeea684ab48ae88ec265</sha1>
+    <cpe>cpe:/a:chromium_project:chromium</cpe>
```
Comment (tisonkun) [inline src/owasp-dependency-check-false-positives.xml:62]:
```
2022-08-25T19:56:04.2775687Z [ERROR] Failed to execute goal org.owasp:dependency-check-maven:7.1.0:aggregate (default) on project pulsar: 
2022-08-25T19:56:04.2776396Z [ERROR] 
2022-08-25T19:56:04.2777027Z [ERROR] One or more dependencies were identified with vulnerabilities that have a CVSS score greater than or equal to '7.0': 
2022-08-25T19:56:04.2777476Z [ERROR] 
2022-08-25T19:56:04.2792695Z [ERROR] netty-tcnative-boringssl-static-2.0.52.Final-linux-x86_64.jar: CVE-2011-1797(9.3)
```

There are still several failures. It seems related to os classfier.

Comment (nicoloboschi) [reply src/owasp-dependency-check-false-positives.xml:62]: @tisonkun yes it is. I added the regex instead of the sha to get all the different classifiers

**Comment by tisonkun** [comment]
cc @codelipenghui @eolivelli we can merge this patch and unblock other patches touching dependencies.

**Review by eolivelli — APPROVED** [review]
+1

**Pull request merged**